about.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: Discovery Environment Docs
+---
+
+The [Discovery Environment (DE)](http://www.cyverse.org/ci/discovery-environment) is the primary 
+web interface and platform to access the powerful computing, storage, and analysis application 
+resources of the [CyVerse](http://www.cyverse.org) cyberinfrastructure.
+
+# Useful Links
+
+* [CyVerse Main Page](http://www.cyverse.org/)
+* [Discovery Environment Main Page](http://www.cyverse.org/ci/discovery-environment)
+* [DE Docker Hub](https://hub.docker.com/r/discoenv/)
+* [DE User Manual](https://pods.iplantcollaborative.org/wiki/display/DEmanual/Table+of+Contents)
+* [Ask iPlant](http://ask.iplantcollaborative.org/questions/)
+* [iPlant Wiki](https://pods.iplantcollaborative.org/wiki/dashboard.action)
+* [iPlant Facebook](https://www.facebook.com/iPlantCollab/)
+* [iPlant Twitter](https://twitter.com/iPlantCollab)
+
+# Other Links
+* [RFC 7519; JSON Web Token](https://tools.ietf.org/html/rfc7519)
+* [RFC 7515; JSON Web Signature](https://tools.ietf.org/html/rfc7515)
+* [jwt.io](http://jwt.io/)
+* [GWT](http://www.gwtproject.org/)
```

Comment:
If we're linking to GWT, we should link to [GXT](https://www.sencha.com/products/gxt/)